src/main/resources/swagger/api-docs.yaml

```diff
@@ -4115,10 +4115,30 @@ definitions:
       ok:
         type: boolean
         description: whether any system(s) need attention
-
+        default: false
```

Comment:
I'm new to swagger, but I'm wondering if these is the right use for the default keyword. From [swagger docs](http://swagger.io/unlocking-the-spec-the-default-keyword/):
```
Common mistakes when using the default value
There are two possible discrepancies when using the default value:

Defining a default value for a container and marking that container as required is somewhat of a contradiction. If the container is required, it will always be sent and the default value will never take effect.

The second case is using the default as a way to mention a sample value for its container. That is not the proper use of the field and can lead to unexpected behavior in some of the Swagger tools. Some containers offer alternative ways to setting a sample value.
```
Since this is a required field, it may not need a default?